exonum-java-light-client/src/main/java/com/exonum/client/package-info.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2019 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * The Exonum light client.
+ * Can be used to sign transactions before sending to blockchain.
```

Comment:
Same as in POM, as its main functionality is to create a transaction (= fill headers, put some payload, and sign) and submit it to Exonum network.